src/cider/nrepl/middleware/format.clj

```diff
@@ -22,15 +23,23 @@
      (assoc acc
             (keyword->symbol kw)
             (walk/postwalk #(cond-> % (string? %) keyword) rule)))
-   fmt/default-indents
+   {}
    indents))
 
+(def ^File default-config-file (File. ".cljfmt.edn"))
+
 (defn format-code-reply
-  [{:keys [code options]}]
-  (let [opts (some-> options
-                     (select-keys [:indents :alias-map])
-                     (update :indents generate-user-indents)
-                     (update :alias-map #(reduce-kv (fn [m k v] (assoc m (name k) v)) {} %)))]
+  [{:keys [code options ^File config-file]
+    :or {config-file default-config-file}}]
+  (let [config-file-opts (when (-> config-file .exists)
+                           (-> config-file slurp edn/read-string))
```

Comment:
TODO: honor `#re` tag https://clojurians.slack.com/archives/CPABC1H61/p1654089774726049

Reply:
> Looks like it's simply  [...]

Indeed! Done.

> Do we want to use the same logic as clojure-lsp and use read-string when it's a .clj file?

It's not particularly documented in https://github.com/clojure-lsp/clojure-lsp/blob/9f0c724bdc69abbb0b8aa78c8cb7f6256cf1dcbb/lib/src/clojure_lsp/feature/format.clj#L29 so I'd rather avoid that complexity / surface area for bugs.

> Also I assume the config-file key will be sent by clients as a string

No, I'd expect clients to send nothing, here we use an `:or`. The option is there merely for testability.

i.e. I'd be inclined to promote one and only one possible filename.

Reply:
> No, I'd expect clients to send nothing, here we use an :or. The option is there merely for testability.

Then perhaps the bit with this internal testable argument could be factored out into a private function and not be exposed to the public API? 

I guess it's not a big deal if Cider / Calva can be relied not to expose any defcustom for the config file and always pass nil to the format op, but it just seems unnecessary.